WordPress/Classes/Extensions/String+RegEx.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+
+// MARK: - String: RegularExpression Helpers
+//
+extension String {
+
+    func matches(regex: String, options: NSRegularExpression.Options = []) -> [NSTextCheckingResult] {
+        let regex = try! NSRegularExpression(pattern: regex, options: options)
```

Comment:
When the regex is a literal I think `try!` is fine, but in other cases I think its a bit risky.   Thoughts?